src/main/kotlin/io/omnitrade/adapters/OrderDepthAdapter.kt

```diff
@@ -0,0 +1,16 @@
+package io.omnitrade.adapters
+
+import com.google.gson.JsonDeserializationContext
+import com.google.gson.JsonDeserializer
+import com.google.gson.JsonElement
+import io.omnitrade.enumerator.order.OrderDepth
+import java.lang.reflect.Type
+
+internal class OrderDepthAdapter: JsonDeserializer<OrderDepth> {
+  override fun deserialize(json: JsonElement?, typeOfT: Type?, context: JsonDeserializationContext?): OrderDepth? {
+    OrderDepth.values().forEach { orderDepth ->
```

Comment:
Can this be done with a `find`?

Reply:
As suggested by @faogustavo , we probably should use `firstOrNull` here, which I think will do the job in a prettier way 💅 
What do you think?